src/sentry/issues/status_change_message.py

```diff
@@ -3,13 +3,15 @@
 from collections.abc import Sequence
 from dataclasses import dataclass
 from typing import TypedDict
+from uuid import uuid4
 
 
 class StatusChangeMessageData(TypedDict):
     fingerprint: Sequence[str]
     project_id: int
     new_status: int
     new_substatus: int | None
+    id: str
```

Comment:
should this be type `UUID`?

Reply:
I think that in dicts, the type should be primitive for serialization. Maybe the dataclass could have a uuid though